charts/ollama/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: ollama
```

Comment:
I'm guessing this was just derived by `helm pull` ing the ollama chart right?  Could we add a make target for that so it's accessible in the future and perhaps we could even have a github actions cron run to do it and pull request if any changes are found.

Reply:
done